kinpy/jacobian.py

```diff
@@ -1,3 +1,4 @@
+from os import link
```

Comment:
This is not used, is it?

Reply:
Year, you're right. I removed this import.